repose-aggregator/components/filters/valkyrie-authorization/src/main/scala/org/openrepose/filters/valkyrieauthorization/ValkyrieAuthorizationFilter.scala

```diff
@@ -136,10 +136,11 @@ class ValkyrieAuthorizationFilter @Inject()(configurationService: ConfigurationS
     def authorizeDevice(permissions: ValkyrieResult, deviceId: Option[String]): ValkyrieResult = {
```

Comment:
Just speaking generally, and with no real suggestion, the nesting and matching going on in this method make it somewhat difficult to follow.